components/insight/SRC/org/openmicroscopy/shoola/env/data/model/ImportableObject.java

```diff
@@ -648,4 +679,44 @@ public void reUpload(List<ImportableFile> files)
 		this.files = files;
 	}
 
+    /**
+     * Get the checksum algorithm, default: 'File-Size-64'
+     * 
+     * @return See above
+     */
+    public String getChecksumAlgorithm() {
+        String option = (String) skipChoices.get(OPTION_CHECKSUMS);
+        return option != null ? option : "File-Size-64";
```

Comment:
Probably good to use the constants themselves where possible, e.g., `omero.model.enums.ChecksumAlgorithmFileSize64.value`.